<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MajorSettingWindow</class>
 <widget class="QMainWindow" name="MajorSettingWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>283</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>考试科目设置</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../rc.qrc">
    <normaloff>:/ico/images/appstu.ICO</normaloff>:/ico/images/appstu.ICO</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tbSubject">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>281</width>
      <height>161</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
    </property>
    <column>
     <property name="text">
      <string>科目编号</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>科目名称</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>148</x>
      <y>170</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>科目名称：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="editID">
    <property name="geometry">
     <rect>
      <x>77</x>
      <y>170</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>170</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>科目编号：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="editName">
    <property name="geometry">
     <rect>
      <x>218</x>
      <y>170</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAdd">
    <property name="geometry">
     <rect>
      <x>47</x>
      <y>201</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>添加</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnEdit">
    <property name="geometry">
     <rect>
      <x>107</x>
      <y>201</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>修改</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDel">
    <property name="geometry">
     <rect>
      <x>167</x>
      <y>201</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>删除</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnExit">
    <property name="geometry">
     <rect>
      <x>227</x>
      <y>201</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../rc.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnExit</sender>
   <signal>clicked()</signal>
   <receiver>MajorSettingWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>250</x>
     <y>222</y>
    </hint>
    <hint type="destinationlabel">
     <x>280</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editID</sender>
   <signal>editingFinished()</signal>
   <receiver>editName</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>110</x>
     <y>180</y>
    </hint>
    <hint type="destinationlabel">
     <x>241</x>
     <y>184</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editName</sender>
   <signal>editingFinished()</signal>
   <receiver>btnAdd</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>236</x>
     <y>179</y>
    </hint>
    <hint type="destinationlabel">
     <x>75</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbSubject</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>btnEdit</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>140</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>132</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
